hs_core/templates/pages/genericresource.html

```diff
@@ -830,14 +830,13 @@ <h4 class="modal-title">Manage access</h4>
 
         {#        ======= Content =======#}
 
+        {% if cm.resource_type != "ToolResource" %}
```

Comment:
@mauriel133 OK. I will defer to you on whether you want to add a new flag or just keep the way it is implemented for now.